packages/utilities/src/dom/raiseClick.ts

```diff
@@ -0,0 +1,7 @@
+/** Raises a click event. */
+export function raiseClick(target: Element): void {
```

Comment:
I was thinking more of like the same `raise` API in EventGroup, or, at the very least, something like `raise(target: Element, eventName: string)` so that it scales across events. Relatively minor but since it's an API change I wanted to call it out. Could do it later too but I think this function would eventually be deprecated.